debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/RecordMakers.java

```diff
@@ -87,22 +218,17 @@ public RecordsForCollection forCollection(CollectionId collectionId) {
         private final SourceInfo source;
         private final Map<String, ?> sourcePartition;
         private final String topicName;
-        private final Schema keySchema;
         private final Schema valueSchema;
         private final Function<Document, String> valueTransformer;
         private final BlockingConsumer<SourceRecord> recorder;
 
         protected RecordsForCollection(CollectionId collectionId, SourceInfo source, String topicName, AvroValidator validator,
-                Function<Document, String> valueTransformer, BlockingConsumer<SourceRecord> recorder) {
+                                       Function<Document, String> valueTransformer, BlockingConsumer<SourceRecord> recorder) {
             this.sourcePartition = source.partition(collectionId.replicaSetName());
             this.collectionId = collectionId;
             this.replicaSetName = this.collectionId.replicaSetName();
             this.source = source;
             this.topicName = topicName;
-            this.keySchema = SchemaBuilder.struct()
```

Comment:
Hey @rhauch, is that what you had in mind, removing the schema for the key altogether? Would that actually work with Avro and the schema registry?